client/src/entrypoints/admin/workflow-action.js

```diff
@@ -121,4 +124,53 @@ function ActivateWorkflowActionsForEditView(formSelector) {
     );
   });
 }
+/** @deprecated RemovedInWagtail70 - Remove global.ActivateWorkflowActionsForEditView usage  */
 window.ActivateWorkflowActionsForEditView = ActivateWorkflowActionsForEditView;
+
+const currentScript = document.currentScript;
```

Comment:
This needs to be accessed outside of the `DOMContentLoaded` event listener as otherwise it would be `null`. See https://developer.mozilla.org/en-US/docs/Web/API/Document/currentScript